src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -151,6 +155,9 @@ public void work() throws Exception {
 		rowCounter.inc();
 		rowMeter.mark();
 
+		if ( scripting != null )
+			scripting.invoke(row);
+
```

Comment:
Should we skip `processRow` if it's suppressed?

Reply:
the thing I'd be worried about is the user shooting themselves in the foot, suppressing everything including maxwell updates, etc.  we have lots of code in the filters to make sure that doesn't happen, but nothing here.

Reply:
👍